examples/pubsub/tutorial_pubsub_subscribe.c

```diff
@@ -57,7 +57,7 @@ addPubSubConnection(UA_Server *server, UA_String *transportProfile,
     if (retval != UA_STATUSCODE_GOOD) {
         return retval;
     }
```

Comment:
Adjust if statement to the open62541 coding conventions.

Reply:
Not sure what the problem here is?

Reply:
We normaly use no brackets for single line if statements ;)